openedx_webhooks/github_views.py

```diff
@@ -46,29 +49,34 @@ def pull_request():
         return "PONG"
 
     pr = event["pull_request"]
+    PR_NUMBER = pr['number']
```

Comment:
I've never seen something like this before, and I don't understand: pr_activity is value that doesn't change during the function, but you gave it a lowercase name.  This isn't a definition of constant that other programmers are going to get.